modules/crypt.cpp

```diff
@@ -43,16 +42,141 @@
 #include <znc/Chan.h>
 #include <znc/User.h>
 #include <znc/IRCNetwork.h>
+#include <openssl/dh.h>
+#include <openssl/bn.h>
+#include <znc/SHA256.h>
 
 #define REQUIRESSL 1
 #define NICK_PREFIX_KEY "[nick-prefix]"
 
 class CCryptMod : public CModule {
+  private:
+    const char *prime1080="FBE1022E23D213E8ACFA9AE8B9DFADA3EA6B7AC7A7B7E95AB5EB2DF858921FEADE95E6AC7BE7DE6ADBAB8A783E7AF7A7FA6A2B7BEB1E72EAE2B72F9FA2BFB2A2EFBEFAC868BADB3E828FA8BADFADA3E4CC1BE7E8AFE85E9698A783EB68FA07A77AB6AD7BEB618ACF9CA2897EB28A6189EFA07AB99A8A7FA9AE299EFA7BA66DEAFEFBEFBF0B7D8B";
```

Comment:
`*` is near type, not near variable. Also spaces missing around `=`. Just use clang-format, there is its configuration in this repo.